packages/shared/fastfetchbot_shared/services/telegraph/__init__.py

```
@@ -0,0 +1,74 @@
+            telegraph_post = await self.telegraph.post(
+                title=self.title,
+                author=self.author,
+                author_url=self.author_url,
+                text=self.content,
+            )
+            logger.info(
+                f"Telegraph: Uploaded to telegraph. Link: {telegraph_post['url']}"
+            )
+            telegraph_url = telegraph_post["url"]
+            return telegraph_url
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)